commons/src/main/java/org/infinispan/commons/configuration/defaults/InfinispanDefaultsResolver.java

```diff
@@ -0,0 +1,63 @@
+package org.infinispan.commons.configuration.defaults;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.infinispan.commons.configuration.attributes.Attribute;
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+
+/**
+ * @author Ryan Emerson
```

Comment:
Could you please remove those empty javadocs? Or even better - put something meaningful here?